src/Storefront/Page/Robots/RobotsPage.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Page\Robots;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Struct\Struct;
+use Shopware\Storefront\Page\Robots\Struct\DomainRuleCollection;
+
+#[Package('storefront')]
+class RobotsPage extends Struct
+{
+    protected DomainRuleCollection $domainRules;
+
+    /**
+     * @var string[]
```

Comment:
```suggestion
     * @var list<string>
```

this would bev more precise. `string[]` can also be `array<string, string>` or `array<int, string>`